.eslintrc.cjs

```diff
@@ -17,10 +20,16 @@ module.exports = {
     'comma-dangle': ['error', 'always-multiline'],
     'no-trailing-spaces': ['error'],
     'eol-last': ['error', 'always'],
+    'node/no-process-env': ['error'],
   },
   overrides: [{
     files: '*.spec.js',
     plugins: ['jest'],
     extends: ['plugin:jest/recommended'],
+    rules: {
+      'node/no-extraneous-import': ['error', {
+        allowModules: ['@jest/globals'],
+      }],
+    },
   }],
```

Comment:
The `overrides` section lets you enable/disable particular rules for only certain files. We only want the `jest` plugin to be used for test files, so we do that here. It's really just an array of more config options, accepting most of the same options as the main config object, plus `files`.

Here we configure [`node/no-extraneous-import` (doc from its plugin package)](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-import.md) to allow us to import `@jest/globals` despite `@jest/globals` not being a package listed in `package.json` (it's a fake package that `jest` inserts when we are running tests, so we don't need to install it, and instead we need to override the rule!).